docs/accounting/default.nix

```diff
@@ -0,0 +1,43 @@
+{ system ? builtins.currentSystem
+, config ? {}
+, pkgs ? import ../../pkgs.nix
+}:
+
+with pkgs;
+
+stdenv.mkDerivation {
+  name = "docsEnv";
+  buildInputs = [ (texlive.combine {
+                    inherit (texlive)
+                      scheme-small
+
+                      # libraries
+                      stmaryrd lm-math amsmath
+                      extarrows cleveref semantic
+                      polytable lazylist
+
+                      # font libraries `mathpazo` seems to depend on palatino, but it isn't pulled.
+                      mathpazo palatino microtype
+
+                      # libraries for marginal notes
+                      xargs todonotes
+
+                      # build tools
+                      latexmk
+
+                      ;
+                  })
+
+                   pkgs.haskellPackages.lhs2tex
+
+                ];
+  src = ./.;
+  buildPhase = "make";
+  installPhase = "mkdir $out; cp *.pdf $out/";
```

Comment:
I just realized that the Haskell file, `accounting/src/Accounting.hs`, is not being copied.  You could change this to
`installPhase = "mkdir $out; cp *.pdf $out/; cp accounting/src/*hs $out";`
but then you would still have to manually copy the file from `results` to where it's supposed to go...